lib/decoding/import_section_parser.ex

```diff
@@ -10,7 +10,7 @@ defmodule WaspVM.Decoder.ImportSectionParser do
 
     entries = if count > 0, do: parse_entries(entries), else: []
 
-    {:imports, entries}
+    {:imports, Enum.reverse(entries)}
```

Comment:
Just a quick note, but we should check to see how many of these types of functions we're using, if were duplicating the function calls, we may be able to reduce execution time through refactoring the data set